ajv/tests/test_questionnaire_validator.js

```diff
@@ -38,10 +38,10 @@ describe("AJV schema Validator", () => {
           res.should.have.status(200);
           const failure = res.body.errors.pop();
           res.body.should.have.property("success", false);
-          failure.should.have.property("keyword", "enum");
+          failure.should.have.property("keyword", "const");
```

Comment:
I know this was already there, but I think it would be worth changing the name of this schema to `test_invalid_block_type.json` because the error is due to the block having `"type": "Test"` as opposed to `"type": "Question"` which I wasn't expecting based off the name

Reply:
Updated 👍